app/src/main/res/layout/fragment_thread_list.xml

```diff
@@ -249,7 +297,7 @@
                                 android:paddingBottom="@dimen/recyclerViewPaddingBottom"
                                 app:behind_swiped_item_icon_margin="@dimen/marginStandard"
                                 app:layout_constraintBottom_toBottomOf="parent"
-                                app:layout_constraintTop_toBottomOf="@id/installUpdateDivider"
+                                app:layout_constraintTop_toBottomOf="@id/webviewVersionDivider"
```

Comment:
Maybe use `@id/webviewVersionGroup` instead ?

Reply:
It's not working for group.